src/core/bittorrent/session.cpp

```diff
@@ -1458,12 +1464,26 @@ void Session::setAppendExtension(bool append)
     }
 }
 
+void Session::networkOnlineStateChanged(const bool online) {
+    Logger::instance()->addMessage(tr("system network status changed to %1", "e.g: system network status changed to ONLINE").arg(online ? "ONLINE" : "OFFLINE"), Log::INFO);
+}
+
+void Session::networkConfigurationChange(const QNetworkConfiguration& cfg) {
+    const QString configured_interface_name = Preferences::instance()->getNetworkInterface();
+    const QString changed_interface = cfg.name();
+    if (configured_interface_name.isEmpty() || configured_interface_name == changed_interface) {
+        Logger::instance()->addMessage(tr("network configuration of %1 has changed, refreshing session binding", "e.g: network configuration of tun0 has changed, refreshing session binding").arg(changed_interface), Log::INFO);
```

Comment:
tr("**N**etwork configuration of **'%1'** has changed, refreshing session binding", "e.g: **N**etwork configuration of **'tun0'** has changed, refreshing session binding")